content/docs/guides/observability/metrics.md

```diff
@@ -126,35 +122,41 @@ annotations:
       target_label: source_workload_name
 ```
 
-### Metrics scraping
+#### Grafana
+
+The following section assumes a Prometheus instance has already been configured as a data source for a running Grafana instance. Refer to the [Prometheus and Grafana](/docs/demos/prometheus_grafana) demo for an example on how to create and configure a Grafana instance.
+
+##### Importing OSM Dashboards
+
+The OSM dashboards (if desired) can be imported through several mechanism to the external Grafana instance.
+OSM Dashboards are available both through:
+- [OSM repository](https://github.com/openservicemesh/osm/tree/release-v0.9/charts/osm/grafana), and are importable as json blobs through the web admin portal
+- or [online at Grafana.com](https://grafana.com/grafana/dashboards/14145).
+
+Detailed instructions for importing OSM dashboards can be found in the [Prometheus and Grafana](/docs/demos/prometheus_grafana) demo. Refer to [OSM Grafana dashboard](#osm-grafana-dashboards) for an overview of the pre-configured dashboards.
+
+## Metrics scraping
 
 Metrics scraping can be configured using the `osm metrics` command. By default, OSM **does not** configure metrics scraping for pods in the mesh. Metrics scraping can be enabled or disabled at namespace scope such that pods belonging to configured namespaces can be enabled or disabled for scraping metrics.
 
 For metrics to be scraped, the following prerequisites must be met:
 
 - The namespace must be a part of the mesh, ie. it must be labeled with the `openservicemesh.io/monitored-by` label with an appropriate mesh name. This can be done using the `osm namespace add` command.
-- A running service able to scrap Prometheus endpoints. OSM provides configuration for an [automatic bringup of Prometheus](#automatic-bring-up); alternatively users can [bring their own Prometheus](#byo-bring-your-own).
+- A running service able to scrape Prometheus endpoints. OSM provides configuration for an [automatic bringup of Prometheus](#automatic-bring-up); alternatively users can [bring their own Prometheus](#prometheus).
```

Comment:
Looks like the link to `#automatic-bring-up` is broken. Should this be `#automatic-provisioning` instead?